pkgs/development/python-modules/sphinxcontrib-moderncmakedomain/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, hatchling
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-moderncmakedomain";
+  version = "3.27.0";
+  disabled = isPy27;
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sphinxcontrib_moderncmakedomain";
+    hash = "sha256-UeJZ6R9Y0XzA+skwf9QBBqpZ1ayqdBiHkD/DZgNh0aE=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+    sphinx
+  ];
+
```

Comment:
Please run the tests https://github.com/scikit-build/moderncmakedomain/tree/main/tests

Reply:
Done!